core/src/main/java/com/mornati/sample/service/PluginService.java

```diff
@@ -67,6 +76,23 @@ public void startFramework() throws BundleException {
     }
   }
 
+  /**
+   * Felix provides an isolated classloader to each bundle. Bundles need to declare what packages they need in there manifest.
+   * If a Bundle needs e.g. something from orgl.slf4j it aither needs to have it as classes itself or another bundle must export this packages.
+   * java.* packages will be provided by the framework from the classloader it was created with. We can modify this, by giving the framework
+   * a list of additional packages, that are provided by the spring boot container (should include e.g.slf4j)
+  * @return
+  */
+  public String findPackageNamesStartingWith() {
```

Comment:
So @juergen-albert is I understand well, in this way we are exposing everything to the bundles, except the filtered packages?

Reply:
Exactly.